research/static/research/js/research.js

```diff
@@ -287,6 +287,7 @@ $(function () {
     chunkSize: 25 * 1024 * 1024,
     forceChunkSize: true,
     simultaneousUploads: 1,
+    testChunks: false,         // Disabled because of need to be able to overwrite files with different content
```

Comment:
We can apply the same fix for the deposit module: https://github.com/UtrechtUniversity/yoda-portal/blob/9cc06c300485c8305bb8387a497cdfa10bf6efab/deposit/static/deposit/js/data.js#L193-L200